src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.cs

```diff
@@ -1045,7 +1045,8 @@ public class Mappings
                        'Get-AzApplicationGatewayUrlPathMapConfig':  'Get-AzureRmApplicationGatewayUrlPathMapConfig',
                        'Get-AzNetworkWatcherConnectionMonitor':  'Get-AzureRmNetworkWatcherConnectionMonitor',
                        'Remove-AzVirtualNetworkSubnetConfig':  'Remove-AzureRmVirtualNetworkSubnetConfig',
-                       'Get-AzBgpServiceCommunity':  'Get-AzureRmBgpServiceCommunity'
+                       'Get-AzBgpServiceCommunity':  'Get-AzureRmBgpServiceCommunity',
+                       'Get-AzInterfaceEndpoint':  'Get-AzInterfaceEndpoint',
```

Comment:
there is something wrong here...
It should be 
'Get-AzInterfaceEndpoint':  'Get-AzureRmInterfaceEndpoint',